Week7/week7_solution.py

```diff
@@ -0,0 +1,192 @@
+Enter the sells of january month : 4500
+Enter the sells of febuary month : 5000
+Enter the sells of march month : 3400
+Enter the sells of apr month : 2300
+Enter the sells of may month : 5000
+Enter the sells of june month : 4500
+Enter product number : 7
+Enter product name : kurti
+Enter the sells of january month : 2300
+Enter the sells of febuary month : 5000
+Enter the sells of march month : 4500
+Enter the sells of apr month : 3400
+Enter the sells of may month : 4600
+Enter the sells of june month : 4500
+Enter product number : 8
+Enter product name : choli
+Enter the sells of january month : 6000
+Enter the sells of febuary month : 7000
+Enter the sells of march month : 5800
+Enter the sells of apr month : 7050
+Enter the sells of may month : 7800
+Enter the sells of june month : 9000
+Enter product number : 9
+Enter product name : sadi
+Enter the sells of january month : 4000
+Enter the sells of febuary month : 5600
+Enter the sells of march month : 4500
+Enter the sells of apr month : 3400
+Enter the sells of may month : 4000
+Enter the sells of june month : 3400
+Enter product number : 10
+Enter product name : nightdress
+Enter the sells of january month : 3000
+Enter the sells of febuary month : 4000
+Enter the sells of march month : 2300
+Enter the sells of apr month : 4000
+Enter the sells of may month : 3400
+Enter the sells of june month : 3500
+Enter product number : 11
+Enter product name : googles
+Enter the sells of january month : 3000
+Enter the sells of febuary month : 2300
+Enter the sells of march month : 4500
+Enter the sells of apr month : 3400
+Enter the sells of may month : 2300
+Enter the sells of june month : 4500
+Enter product number : 12
+Enter product name : shoes
+Enter the sells of january month : 3000
+Enter the sells of febuary month : 3400
+Enter the sells of march month : 3500
+Enter the sells of apr month : 4500
+Enter the sells of may month : 3000
+Enter the sells of june month : 2300
+
+Create CSV file for product selling for 6 Months ( Prod_No | Prod_Name | Jan | Feb | Mar | Apr | May | Jun) for 5 products.
+-->
+	import csv
+	with open('C:\\22bca62\\python\\product.csv','w',newline='')as f:
+                 obj=csv.writer(f)
+                 obj.writerow(h)
+                 obj.writerows(l)
+
+Perform following operations.
+
+1. Add 12 Records. Take input from user.
+2. Create dataframe.
+-->
+	import pandas as pd
+	df=pd.read_csv('C:\\22bca62\\python\\product.csv')
+	df
+	
+3. Change Column Name Product No, Product Name, January, February, March, April, May, June.
+-->
+	df.columns = ["Product No", "Product Name", "January", "February", "March", "April", "May", "June"]
+
+4. Add column "Total Sell" to count total of all month and "Average Sell"
+-->
+	df["Total Sell"] = df.iloc[:, 2:].sum(axis=1)
+	df["Average Sell"] = df.iloc[:, 2:8].mean(axis=1)
+
+5. Add 2 row at the end.
+-->
+	df = df.append({"Product No": "13", "Product Name": "abc", "January": 100, "February": 200, "March": 300,

```

Comment:
are you sure append works? as per my knowledge append doesnt work as attribute for DataFrame